test/e2e/framework/addon/vault/vault.go

```diff
@@ -105,44 +106,136 @@ func (v *Vault) Setup(cfg *config.Config) error {
 	}
 	v.details.Kubectl = cfg.Kubectl
 	v.chart = &chart.Chart{
-		Base:        v.Base,
-		ReleaseName: "chart-vault-" + v.Name,
-		Namespace:   v.Namespace,
-		ChartName:   cfg.RepoRoot + "/test/e2e/charts/vault",
```

Comment:
Can we remove this chart now?
Does someone know why we were maintaining our own helm chart?

Reply:
yes, I just forgot about it.